i18n/zh/docusaurus-plugin-content-docs/current/kubeedge.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 为什么选择KubeEdge
+title: 为什么选择 KubeEdge
```

Comment:
Why add spaces here?

Reply:
I got used to it personally, I'll make the necessary changes right away